includes/Modules/AdSense.php

```diff
@@ -421,6 +422,20 @@ protected function parse_data_response( Data_Request $data, $response ) {
 		return parent::parse_data_response( $data, $response );
 	}
 
+	/**
+	 * Checks for the state of an Account, whether close or not. 
+	 * 
+	 * @since 1.71.0
+	 * 
+	 * @param account array
+	 * 
+	 * @return boolean
+	 */
+	public static function is_account_not_closed($account){
+
+		return $account.state == 'CLOSED';
```

Comment:
This isn't valid PHP and will cause a fatal error. It needs to use `$account->state`, or even better `$account->getState()`.

Also same like above, please review the WordPress coding standards. This needs to use a strict comparison and Yoda condition.